### PR TITLE
Add sendTyping and sendReceipt to dbus interface

### PR DESCRIPTION
**Pull request by JtheSaw**
Add DBUS implementation and documentation for both methods:
- sendTyping
- sendReceipt

### CODE REVIEW

File: src/main/java/org/asamk/Signal.java [hunk 1]
```diff
@@ -21,6 +21,14 @@ long sendMessage(
             String message, List<String> attachments, List<String> recipients
     ) throws Error.AttachmentInvalid, Error.Failure, Error.InvalidNumber, Error.UntrustedIdentity;
 
+    void sendTyping(
+            String recipient, Boolean stop
```
Comment (AsamK) [inline src/main/java/org/asamk/Signal.java:25]:
```suggestion
            String recipient, boolean stop
```

File: src/main/java/org/asamk/Signal.java [hunk 2]
```diff
@@ -21,6 +21,14 @@ long sendMessage(
...
+    ) throws Error.Failure, Error.GroupNotFound, Error.UntrustedIdentity;
+
+    void sendReceipt(
```
Comment (AsamK) [inline src/main/java/org/asamk/Signal.java:28]: Should be renamed to `sendReadReceipt`

File: src/main/java/org/asamk/Signal.java [hunk 3]
```diff
@@ -21,6 +21,14 @@ long sendMessage(
...
+            String recipient, long targetSentTimestamp
```
Comment (AsamK) [inline src/main/java/org/asamk/Signal.java:29]: Should accept a `List<Long>` of timeframe timestamps to mark multiple messages as read at the same time

**Review by AsamK — CHANGES_REQUESTED** [review]

**Comment by AsamK** [comment]
Thanks, please adapt the documentation to the review changes as well.

**Comment by JtheSaw** [comment]
`List<Long>` must a new type, right?
`* <ax>  : Array of signed 64 bit integer`

**Review by AsamK — APPROVED** [review]

**Pull request merged**